packages/typeschema-validator/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@hono/schema-validator",
+  "version": "0.0.0",
+  "description": "Validator middleware for multiple schema validation libraries based on schema.com",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "test": "NODE_OPTIONS=\"$NODE_OPTIONS --experimental-vm-modules\" jest",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
```

Comment:
Can you use `tsup` to build and `vitest` for testing, referring to the `hello` middleware?

https://github.com/honojs/middleware/tree/main/packages/hello